changelogs/unreleased/000-yvespp

```diff
@@ -0,0 +1,9 @@
+## All changes
```

Comment:
Could you refine the changelog and rename the file name with the correct PR number?